website/notifications/model.py

```diff
@@ -4,18 +4,35 @@
 from modularodm.exceptions import ValidationValueError
 
 from website.project.model import Node
-from website.notifications.constants import NOTIFICATION_TYPES
+from website.notifications.constants import NOTIFICATION_TYPES, NODE_SUBSCRIPTIONS_AVAILABLE, USER_SUBSCRIPTIONS_AVAILABLE
+import re
 
 
 def validate_subscription_type(value):
     if value not in NOTIFICATION_TYPES:
         raise ValidationValueError
 
 
+def validate_event_type(value):
+    """
+
+    :param value:
+
+    Checks if the value passed in is in the NODE_SUBSCRIPTIONS_AVAILABLE or USER_SUBSCRIPTIONS_AVAILABLE
+    if it is an event name.  Since the only things from NODE_SUBSCRIPTIONS_AVAILABLE that contains "file_updated"
+    is preceded by a user ID, a regex checker was added for that.
+    """
+    prefix = re.compile('[[a-zA-Z0-9]*_]?')
```

Comment:
I'm not convinced that this is exactly doing what you want it to, and we'd need some tests for this. What are some examples of matching strings?